src/main/java/org/traccar/model/Position.java

```diff
@@ -90,6 +90,9 @@
     public static final String KEY_DTCS = "dtcs";
     public static final String KEY_OBD_SPEED = "obdSpeed"; // knots
     public static final String KEY_OBD_ODOMETER = "obdOdometer"; // meters
+    public static final String KEY_OBD_IGNITION_TIMING = "obdIgnitionTiming"; // degree
+    public static final String KEY_OBD_INTAKE_FLOW_RATE = "obdIntakeFlowRate"; // %
+
```

Comment:
We don't add these unless they are used in multiple protocols. Please use strings directly instead.

Reply:
Noted.